src/main/java/com/voorbeeld/TechItEasy/model/Television.java

```diff
@@ -3,14 +3,14 @@
 public class Television {
     private String name;
     private int price;
-    private int id;
+    private String brand;
 
```

Comment:
Waarom heb je je id nou weg gehaalt uit de class?

In je controller gebruik je het id nu om een bepaalde index in je array te targeten,

Maar wat je wilt is dat je /{id} matcht met het id van je object.